### PR TITLE
Minor schema changes from review comments and for better compatibility

### PR DESCRIPTION
**Pull request by yao-msft**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/759)

### CODE REVIEW

**Review by JohnMcPMS — APPROVED** [review]
I'm fine with these; did you get sign-off from other via mail/Teams or should they officially weigh in here?

**Comment by yao-msft** [comment]
I'll merge after consensus is reached

**Pull request merged**
